### DISCUSSION TITLE
使用AI提取历年漏洞Fofa查询语句 - 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#30] [General]
# 使用AI提取历年漏洞Fofa查询语句 - 倾旋的博客

https://payloads.online/archivers/2024-09-08/extract-vuln-search-queries/

**Comment by 80999348** [comment]
能给个[Markdown]格式的文档例子吗？

**Reply by Rvn0xsy** [reply]
什么意思，我不太理解
